markdown/org/docs/patterns/waralee/cutting/en.md

```diff
@@ -1,18 +1,37 @@
-Waralee consists of two identical pieces of fabric.
+<Note>
+
+Waralee pants parts consists of two mirrored pieces of fabric.
 They are basically big rectangles with pieces missing.
-As such, it is easier to print out the piece you need to cut out and measure the rest,
+As such, it is easier to print out the part you need to cut out and measure the rest,
```

Comment:
I guess the idea is for this sentence to agree with the earlier one with regard to the usage of "piece" or "part".

Reply:
part for pattern